src/metabase/util/humanization.cljc

```diff
@@ -41,9 +41,3 @@
 (defmethod name->human-readable-name :none
   [_strategy s]
   s)
-
-;;; `:advanced` doesn't exist anymore, it used to be super fancy and do neat things. On the off chance someone still
-;;; tries to use it, just do the same thing `:simple` does.
-(defmethod name->human-readable-name :advanced
-  [strategy s]
-  ((get-method name->human-readable-name :simple) strategy s))
```

Comment:
@noahmoss Removing this per your suggestion. Could you sanity check this decision? I don't know the whole context of our cljc files, and don't know if somehow some other process might invoke this, expecting `:advanced` to be supported.

Reply:
Makes sense to me